packages/web-app/app/_ui/project/contribution/DataFields.tsx

```diff
@@ -36,61 +25,49 @@ const LoadingField = () => (
 );
 
 export const DataFields = () => {
-  const { data: maxContribution } = useReadCtzndSaleMaxContribution();
+  const { data: totalTokensForSale } = useReadCtzndSaleTotalTokensForSale();
   const { data: minContribution } = useReadCtzndSaleMinContribution();
```

Comment:
We need to call `useCtzndSaleTokenToPaymentToken` to convert this value to USDC. The amount returned by this is in CTND tokens